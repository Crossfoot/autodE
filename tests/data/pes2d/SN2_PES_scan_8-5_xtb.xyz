6

F   -2.85189  -0.16689  0.04960  
Cl  2.02532   0.11838   -0.03246 
C   -0.95192  -0.05526  0.01749  
H   -0.76909  -0.62259  -0.87263 
H   -0.74392  -0.52353  0.95825  
H   -0.85145  1.00981   -0.04157 
