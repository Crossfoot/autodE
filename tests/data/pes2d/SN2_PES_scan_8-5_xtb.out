      -----------------------------------------------------------      
     |                   =====================                   |     
     |                           x T B                           |     
     |                   =====================                   |     
     |                         S. Grimme                         |     
     |          Mulliken Center for Theoretical Chemistry        |     
     |                    University of Bonn                     |     
      -----------------------------------------------------------      

   * xtb version 6.2.3 (830e466) compiled by 'ehlert@majestix' on 2020-03-16

   xtb is free software: you can redistribute it and/or modify it under
   the terms of the GNU Lesser General Public License as published by
   the Free Software Foundation, either version 3 of the License, or
   (at your option) any later version.
   
   xtb is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU Lesser General Public License for more details.
   
   Cite this work as:
   * S. Grimme, C. Bannwarth, P. Shushkov, J. Chem. Theory Comput., 2017,
     13, 1989-2009. DOI: 10.1021/acs.jctc.7b00118
   * C. Bannwarth, S. Ehlert and S. Grimme., J. Chem. Theory Comput., 2019,
     15, 1652-1671. DOI: 10.1021/acs.jctc.8b01176
   * P. Pracht, E. Caldeweyher, S. Ehlert, S. Grimme, ChemRxiv, 2019, preprint.
     DOI: 10.26434/chemrxiv.8326202.v1
   
   for DFT-D4:
   * E. Caldeweyher, C. Bannwarth and S. Grimme, J. Chem. Phys., 2017,
     147, 034112. DOI: 10.1063/1.4993215
   * E. Caldeweyher, S. Ehlert, A. Hansen, H. Neugebauer, S. Spicher,
     C. Bannwarth and S. Grimme, J. Chem. Phys., 2019, 150, 154122.
     DOI: 10.1063/1.5090222
   
   for sTDA-xTB:
   * S. Grimme and C. Bannwarth, J. Chem. Phys., 2016, 145, 054103.
     DOI: 10.1063/1.4959605
   
   in the mass-spec context:
   * V. Asgeirsson, C. Bauer and S. Grimme, Chem. Sci., 2017, 8, 4879.
     DOI: 10.1039/c7sc00601b
   
   for metadynamics refer to:
   * S. Grimme, J. Chem. Theory Comput., 2019, 155, 2847-2862
     DOI: 10.1021/acs.jctc.9b00143
   
   with help from (in alphabetical order)
   C. Bannwarth, F. Bohle, G. Brandenburg, E. Caldeweyher, M. Checinski,
   S. Dohm, S. Ehlert, S. Ehrlich, F. März, H. Neugebauer, J. Pisarek,
   P. Pracht, P. Shushkov, and S. Spicher.
   
 * started run on 2020/04/08 at 23:04:53.746     

           -------------------------------------------------
          |                Calculation Setup                |
           -------------------------------------------------

          program call               : /home/tom/.local/bin/xtb SN2_PES_scan_8-5_xtb.xyz --chrg -1 --opt --input xcontrol_SN2_PES_scan_8-5
          coordinate file            : SN2_PES_scan_8-5_xtb.xyz
          omp threads                :                     1
          number of atoms            :                     6
          number of electrons        :                    22
          charge                     :                    -1
          spin                       :                   0.0
          first test random number   :      0.78850904480706

constraining bond 1 3 to    1.6050000 Å, actual value:    1.9035174 Å
constraining bond 2 3 to    3.2830000 Å, actual value:    2.9827175 Å

molecular fragmentation (1/2 indicates fragments):
122222
# atoms in fragment 1/2:     1     5
 fragment masses (1/2) :       19.00       50.49
CMA distance (Bohr)    :   7.574
constraining FC (au)   : 10.0000
########################################################################
[WARNING] Please study the warnings concerning your input carefully
-2- prog_main: XTBPATH is not set, using XTBHOME instead
-1- prog_main: XTBHOME is not set, using HOME instead
########################################################################
           ------------------------------------------------- 
          |                 G F N 2 - x T B                 |
          | Geometry, Frequencies, Noncovalent interactions |
          |            JCTC 2019 parametrisation            |
           ------------------------------------------------- 
             k(s)              :                1.8500
             k(p)              :                2.2300
             k(d)              :                2.2300
             k(f)              :                2.0000
             kEN (H0ij)        :               -2.0000
             D4 a1             :                0.5200
             D4 a2             :                5.0000
             D4 s6             :                1.0000
             D4 s8             :                2.7000
             D4 s9             :                5.0000
             alphaj            :                2.0000

  Z AO/shell   Hii/eV     exponent
  1     Wed Apr 25 08:07:45 CEST 2018   EN: 2.200 GM2: 0.406  GM3: 0.0800  RAES: 1.40
     1s    -10.707211    1.230000
  6     Wed Apr 25 08:07:45 CEST 2018   EN: 2.550 GM2: 0.538  GM3: 0.1500  RAES: 3.00
     2s    -13.970922    2.096432
     2p    -10.063292    1.800000
  9     Mon Apr 23 13:58:50 CEST 2018   EN: 3.980 GM2: 0.532  GM3: 0.1426  RAES: 2.40
     2s    -23.458179    2.416361
     2p    -15.746583    2.308399
 17     Mon Apr 23 14:09:22 CEST 2018   EN: 3.160 GM2: 0.249  GM3: 0.1495  RAES: 2.50
     3s    -29.278781    2.485265
     3p    -12.673758    2.199650
     3d     -0.240338    2.476089

           ------------------------------------------------- 
          |        Self-Consistent Charge Iterations        |
           ------------------------------------------------- 

          ...................................................
          :                      SETUP                      :
          :.................................................:
          :  # basis functions                  21          :
          :  # atomic orbitals                  20          :
          :  # shells                           10          :
          :  # electrons                        22          :
          :  max. iterations                   250          :
          :  Hamiltonian                  GFN2-xTB          :
          :  restarted?                      false          :
          :  GBSA solvation                  false          :
          :  PC potential                    false          :
          :  electronic temp.          300.0000000     K    :
          :  accuracy                    1.0000000          :
          :  -> integral cutoff          0.2500000E+02      :
          :  -> integral neglect         0.1000000E-07      :
          :  -> SCF convergence          0.1000000E-05 Eh   :
          :  -> wf. convergence          0.1000000E-03 e    :
          :  Broyden damping             0.4000000          :
          ...................................................

 iter      E             dE          RMSdq      gap      omega  full diag
   1    -13.1018526 -0.131019E+02  0.675E+00    4.71       0.0  T
   2    -13.1474898 -0.456372E-01  0.314E+00    2.11       1.0  T
   3    -13.1855548 -0.380651E-01  0.298E+00    1.95       1.0  T
   4    -13.1962010 -0.106462E-01  0.108E+00    1.01       1.0  T
   5    -13.1981508 -0.194974E-02  0.102E+00    1.78       1.0  T
   6    -13.2002591 -0.210828E-02  0.222E-01    1.51       1.0  T
   7    -13.2003520 -0.928966E-04  0.816E-02    1.43       1.0  T
   8    -13.2003535 -0.149743E-05  0.811E-02    1.40       1.0  T
   9    -13.2003727 -0.192795E-04  0.179E-02    1.44       2.3  T
  10    -13.2003736 -0.849833E-06  0.115E-02    1.44       3.6  T
  11    -13.2003739 -0.344089E-06  0.135E-03    1.43      30.3  T
  12    -13.2003739 -0.432492E-08  0.497E-04    1.43      82.2  T
  13    -13.2003739 -0.794762E-09  0.313E-05    1.43    1305.6  T

   *** convergence criteria satisfied after 13 iterations ***

         #    Occupation            Energy/Eh            Energy/eV
      -------------------------------------------------------------
         1        2.0000           -0.8073926             -21.9703
       ...           ...                  ...                  ...
         5        2.0000           -0.3641936              -9.9102
         6        2.0000           -0.3189614              -8.6794
         7        2.0000           -0.2844760              -7.7410
         8        2.0000           -0.2844744              -7.7409
         9        2.0000           -0.2048588              -5.5745
        10        2.0000           -0.2011569              -5.4738
        11        2.0000           -0.2011567              -5.4738 (HOMO)
        12                         -0.1484900              -4.0406 (LUMO)
        13                          0.2588090               7.0426
        14                          0.3125467               8.5048
        15                          0.3125475               8.5048
        16                          0.3128840               8.5140
       ...                                ...                  ...
        20                          0.3959276              10.7737
      -------------------------------------------------------------
                  HL-Gap            0.0526667 Eh            1.4331 eV
             Fermi-level           -0.1745027 Eh           -4.7485 eV

 SCC (total)                   0 d,  0 h,  0 min,  0.003 sec
 SCC setup                      ...        0 min,  0.000 sec (  2.288%)
 Dispersion                     ...        0 min,  0.000 sec (  0.484%)
 integral evaluation            ...        0 min,  0.001 sec ( 24.397%)
 zeroth order Hamiltonian       ...        0 min,  0.000 sec (  0.554%)
 iterations                     ...        0 min,  0.001 sec ( 44.096%)
 molecular gradient             ...        0 min,  0.001 sec ( 23.359%)
 printout                       ...        0 min,  0.000 sec (  4.100%)

         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         ::                     SUMMARY                     ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         :: total energy              -6.739768740966 Eh    ::
         :: gradient norm             27.773422321570 Eh/a0 ::
         :: HOMO-LUMO gap              1.433132749215 eV    ::
         ::.................................................::
         :: SCC energy               -13.200373938666 Eh    ::
         :: -> isotropic ES            0.169104693453 Eh    ::
         :: -> anisotropic ES         -0.001552217517 Eh    ::
         :: -> anisotropic XC          0.004950236818 Eh    ::
         :: -> dispersion             -0.001845042619 Eh    ::
         :: repulsion energy           0.058324369315 Eh    ::
         :: add. restraining           6.402281350076 Eh    ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::

      ----------------------------------------------------------- 
     |                   =====================                   |
     |                        A N C O P T                        |
     |                   =====================                   |
     |               Approximate Normal Coordinate               |
     |                Rational Function Optimizer                |
      ----------------------------------------------------------- 

          ...................................................
          :                      SETUP                      :
          :.................................................:
          :   optimization level            normal          :
          :   max. optcycles                   200          :
          :   ANC micro-cycles                  20          :
          :   degrees of freedom                12          :
          :.................................................:
          :   RF solver                   davidson          :
          :   write xtbopt.log                true          :
          :   linear?                        false          :
          :   energy convergence         0.5000000E-05 Eh   :
          :   grad. convergence          0.1000000E-02 Eh/α :
          :   maximium RF displ.         1.0000000          :
          :   Hlow (freq-cutoff)         0.1000000E-01      :
          :   Hmax (freq-cutoff)         5.0000000          :
          :   S6 in model hess.         20.0000000          :
          ...................................................

generating ANC from model Hessian ...
Using Lindh-Hessian (1995)
 Shifting diagonal of input Hessian by   0.566273706528572     
 Lowest  eigenvalues of input Hessian
    0.010000    0.010046   -0.000000   -0.000000   -0.000000    0.000000
    0.000000    0.000000    0.606714    0.705305    0.705390    1.012390
    1.256168    1.256444    4.419512    4.419798   20.620237   60.787487
 Highest eigenvalues
    1.256168    1.256444    4.419512    4.419798   20.620237   60.787487
 

........................................................................
.............................. CYCLE    1 ..............................
........................................................................
   1    -13.2003739 -0.132004E+02  0.243E-05    1.43       0.0  T
   2    -13.2003739 -0.591527E-12  0.178E-05    1.43    2289.1  T
   3    -13.2003739 -0.143530E-11  0.452E-06    1.43    9035.9  T
     SCC iter.                  ...        0 min,  0.000 sec
     gradient                   ...        0 min,  0.001 sec
 * total energy  :    -6.7397687 Eh     change       -0.9075904E-09 Eh
   gradient norm :    27.7734221 Eh/α   predicted     0.0000000E+00 (-100.00%)
   displ. norm   :     0.9140292 α      lambda       -0.2538572E+02
   maximum displ.:     0.9140247 α      in ANC's #11, #12, #6, ...

........................................................................
.............................. CYCLE    2 ..............................
........................................................................
   1    -13.1839473 -0.131839E+02  0.261E+00   10.11       0.0  T
   2    -13.1986712 -0.147239E-01  0.135E+00    9.30       1.0  T
   3    -13.2000994 -0.142814E-02  0.731E-01    9.13       1.0  T
   4    -13.2002294 -0.130025E-03  0.292E-01    8.89       1.0  T
   5    -13.2002584 -0.289702E-04  0.300E-02    8.85       1.4  T
   6    -13.2002590 -0.613424E-06  0.177E-02    8.85       2.3  T
   7    -13.2002592 -0.211068E-06  0.653E-03    8.85       6.2  T
   8    -13.2002593 -0.743671E-07  0.124E-03    8.85      33.0  T
   9    -13.2002593 -0.356120E-08  0.415E-04    8.85      98.3  T
     SCC iter.                  ...        0 min,  0.001 sec
     gradient                   ...        0 min,  0.001 sec
 * total energy  :    -7.0333695 Eh     change       -0.2936007E+00 Eh
   gradient norm :    27.2410668 Eh/α   predicted    -0.2329710E+02 (7834.96%)
   displ. norm   :     0.3854054 α      lambda       -0.1049825E+02
   maximum displ.:     0.3853740 α      in ANC's #11, #3, #6, ...

........................................................................
.............................. CYCLE    3 ..............................
........................................................................
   1    -13.2103233 -0.132103E+02  0.115E+00    3.69       0.0  T
   2    -13.2121469 -0.182361E-02  0.631E-01    4.03       1.0  T
   3    -13.2124320 -0.285142E-03  0.312E-01    4.32       1.0  T
   4    -13.2124465 -0.145043E-04  0.121E-01    4.37       1.0  T
   5    -13.2124570 -0.104523E-04  0.338E-02    4.38       1.2  T
   6    -13.2124581 -0.109218E-05  0.149E-02    4.38       2.7  T
   7    -13.2124584 -0.383166E-06  0.691E-03    4.39       5.9  T
   8    -13.2124585 -0.754049E-07  0.651E-04    4.39      62.7  T
   9    -13.2124585 -0.425979E-09  0.182E-04    4.39     223.8  T
     SCC iter.                  ...        0 min,  0.001 sec
     gradient                   ...        0 min,  0.001 sec
 * total energy  :   -13.0394853 Eh     change       -0.6006116E+01 Eh
   gradient norm :     3.9636506 Eh/α   predicted    -0.6028817E+01 (   0.38%)
   displ. norm   :     0.0795310 α      lambda       -0.2618012E+00
   maximum displ.:     0.0653399 α      in ANC's #11, #6, #3, ...

........................................................................
.............................. CYCLE    4 ..............................
........................................................................
   1    -13.2183627 -0.132184E+02  0.163E-01    3.63       0.0  T
   2    -13.2184104 -0.476743E-04  0.994E-02    3.68       1.0  T
   3    -13.2184170 -0.664301E-05  0.539E-02    3.73       1.0  T
   4    -13.2184177 -0.692473E-06  0.225E-02    3.74       1.8  T
   5    -13.2184180 -0.329356E-06  0.516E-03    3.74       7.9  T
   6    -13.2184181 -0.276706E-07  0.171E-03    3.74      23.9  T
     SCC iter.                  ...        0 min,  0.000 sec
     gradient                   ...        0 min,  0.001 sec
 * total energy  :   -13.1708482 Eh     change       -0.1313628E+00 Eh
   gradient norm :     0.2052299 Eh/α   predicted    -0.1317287E+00 (   0.28%)
   displ. norm   :     0.0599226 α      lambda       -0.9528527E-02
   maximum displ.:     0.0396086 α      in ANC's #12, #3, #6, ...

........................................................................
.............................. CYCLE    5 ..............................
........................................................................
   1    -13.2214726 -0.132215E+02  0.687E-02    3.39       0.0  T
   2    -13.2214784 -0.584714E-05  0.430E-02    3.41       1.0  T
   3    -13.2214795 -0.106272E-05  0.192E-02    3.43       2.1  T
   4    -13.2214796 -0.599642E-07  0.648E-03    3.43       6.3  T
   5    -13.2214796 -0.488580E-07  0.306E-03    3.43      13.3  T
   6    -13.2214796 -0.122999E-07  0.952E-04    3.43      42.9  T
     SCC iter.                  ...        0 min,  0.000 sec
     gradient                   ...        0 min,  0.001 sec
 * total energy  :   -13.1642390 Eh     change        0.6609126E-02 Eh
   gradient norm :     0.6069430 Eh/α   predicted    -0.4781373E-02 (-172.35%)
   displ. norm   :     0.0323313 α      lambda       -0.1748319E-01
   maximum displ.:     0.0285050 α      in ANC's #12, #3, #6, ...

........................................................................
.............................. CYCLE    6 ..............................
........................................................................
   1    -13.2247147 -0.132247E+02  0.504E-02    3.67       0.0  T
   2    -13.2247153 -0.543556E-06  0.301E-02    3.67       1.4  T
   3    -13.2247155 -0.223280E-06  0.507E-03    3.65       8.0  T
   4    -13.2247155 -0.233546E-07  0.264E-03    3.66      15.4  T
     SCC iter.                  ...        0 min,  0.000 sec
     gradient                   ...        0 min,  0.001 sec
 * total energy  :   -13.1734430 Eh     change       -0.9203941E-02 Eh
   gradient norm :     0.0344352 Eh/α   predicted    -0.8750737E-02 (  -4.92%)
   displ. norm   :     0.0328116 α      lambda       -0.6546645E-03
   maximum displ.:     0.0276788 α      in ANC's #3, #6, #12, ...

........................................................................
.............................. CYCLE    7 ..............................
........................................................................
   1    -13.2280501 -0.132281E+02  0.240E-02    3.66       0.0  T
   2    -13.2280507 -0.587318E-06  0.174E-02    3.66       2.3  T
   3    -13.2280508 -0.857942E-07  0.752E-03    3.67       5.4  T
   4    -13.2280508 -0.348001E-08  0.303E-03    3.67      13.5  T
   5    -13.2280508 -0.619554E-08  0.298E-04    3.67     137.2  T
     SCC iter.                  ...        0 min,  0.000 sec
     gradient                   ...        0 min,  0.001 sec
 * total energy  :   -13.1739588 Eh     change       -0.5158073E-03 Eh
   gradient norm :     0.0155719 Eh/α   predicted    -0.3276831E-03 ( -36.47%)
   displ. norm   :     0.0565577 α      lambda       -0.6582226E-03
   maximum displ.:     0.0512232 α      in ANC's #3, #6, #2, ...

........................................................................
.............................. CYCLE    8 ..............................
........................................................................
   1    -13.2325361 -0.132325E+02  0.393E-02    3.65       0.0  T
   2    -13.2325387 -0.261868E-05  0.292E-02    3.65       1.4  T
   3    -13.2325390 -0.299477E-06  0.166E-02    3.66       2.5  T
   4    -13.2325390 -0.273119E-07  0.594E-03    3.66       6.9  T
   5    -13.2325390 -0.154361E-07  0.361E-04    3.66     112.9  T
   6    -13.2325390 -0.920597E-10  0.112E-04    3.66     364.1  T
     SCC iter.                  ...        0 min,  0.000 sec
     gradient                   ...        0 min,  0.001 sec
 * total energy  :   -13.1743202 Eh     change       -0.3614488E-03 Eh
   gradient norm :     0.0100019 Eh/α   predicted    -0.3301569E-03 (  -8.66%)
   displ. norm   :     0.0270712 α      lambda       -0.6454203E-04
   maximum displ.:     0.0202999 α      in ANC's #2, #3, #1, ...

........................................................................
.............................. CYCLE    9 ..............................
........................................................................
   1    -13.2326103 -0.132326E+02  0.302E-02    3.66       0.0  T
   2    -13.2326108 -0.477610E-06  0.183E-02    3.65       2.2  T
   3    -13.2326108 -0.181947E-07  0.734E-03    3.66       5.6  T
   4    -13.2326108 -0.289725E-07  0.124E-03    3.66      33.0  T
   5    -13.2326108 -0.124969E-08  0.471E-04    3.66      86.6  T
     SCC iter.                  ...        0 min,  0.000 sec
     gradient                   ...        0 min,  0.001 sec
 * total energy  :   -13.1743547 Eh     change       -0.3441824E-04 Eh
   gradient norm :     0.0078991 Eh/α   predicted    -0.3229336E-04 (  -6.17%)
   displ. norm   :     0.0840711 α      lambda       -0.2865188E-03
   maximum displ.:     0.0659494 α      in ANC's #3, #2, #1, ...

........................................................................
.............................. CYCLE   10 ..............................
........................................................................
   1    -13.2319132 -0.132319E+02  0.932E-02    3.63       0.0  T
   2    -13.2319244 -0.111375E-04  0.526E-02    3.63       1.0  T
   3    -13.2319252 -0.797328E-06  0.286E-02    3.63       1.4  T
   4    -13.2319253 -0.175121E-06  0.484E-03    3.64       8.4  T
   5    -13.2319254 -0.188413E-07  0.195E-03    3.63      21.0  T
   6    -13.2319254 -0.636034E-08  0.629E-04    3.63      65.0  T
     SCC iter.                  ...        0 min,  0.000 sec
     gradient                   ...        0 min,  0.001 sec
 * total energy  :   -13.1743219 Eh     change        0.3280562E-04 Eh
   gradient norm :     0.0214891 Eh/α   predicted    -0.1442714E-03 (-539.78%)
   displ. norm   :     0.0472041 α      lambda       -0.2235062E-03
   maximum displ.:     0.0364383 α      in ANC's #3, #2, #1, ...

........................................................................
.............................. CYCLE   11 ..............................
........................................................................
   1    -13.2314346 -0.132314E+02  0.473E-02    3.65       0.0  T
   2    -13.2314371 -0.253803E-05  0.282E-02    3.65       1.4  T
   3    -13.2314373 -0.195302E-06  0.145E-02    3.65       2.8  T
   4    -13.2314374 -0.455988E-07  0.297E-03    3.65      13.7  T
   5    -13.2314374 -0.609538E-08  0.709E-04    3.65      57.6  T
     SCC iter.                  ...        0 min,  0.000 sec
     gradient                   ...        0 min,  0.001 sec
 * total energy  :   -13.1744395 Eh     change       -0.1176755E-03 Eh
   gradient norm :     0.0026085 Eh/α   predicted    -0.1120010E-03 (  -4.82%)
   displ. norm   :     0.0052825 α      lambda       -0.3453841E-05
   maximum displ.:     0.0035357 α      in ANC's #1, #2, #6, ...

........................................................................
.............................. CYCLE   12 ..............................
........................................................................
   1    -13.2311181 -0.132311E+02  0.492E-03    3.65       0.0  T
   2    -13.2311181 -0.754431E-08  0.288E-03    3.65      14.2  T
   3    -13.2311181 -0.771873E-09  0.933E-04    3.65      43.8  T
   4    -13.2311181 -0.502512E-09  0.271E-04    3.65     150.8  T
     SCC iter.                  ...        0 min,  0.000 sec
     gradient                   ...        0 min,  0.001 sec
 * total energy  :   -13.1744410 Eh     change       -0.1428090E-05 Eh
   gradient norm :     0.0009997 Eh/α   predicted    -0.1725334E-05 (  20.81%)
   displ. norm   :     0.0245459 α      lambda       -0.5766874E-05
   maximum displ.:     0.0214427 α      in ANC's #1, #2, #6, ...

   *** GEOMETRY OPTIMIZATION CONVERGED AFTER 12 ITERATIONS ***

------------------------------------------------------------------------
 total energy gain   :        -6.4346722 Eh    -4037.8178 kcal/mol
 total RMSD          :         0.2100236 a0        0.1111 Å
 total power (kW/mol):     -1407.8524668 (step)********** (real)
------------------------------------------------------------------------

 ANCopt (total)                0 d,  0 h,  0 min,  0.021 sec
 optimizer setup                ...        0 min,  0.000 sec (  0.259%)
 model hessian                  ...        0 min,  0.000 sec (  0.376%)
 ANC generation                 ...        0 min,  0.000 sec (  0.300%)
 coordinate transformation      ...        0 min,  0.000 sec (  0.226%)
 single point calculation       ...        0 min,  0.020 sec ( 94.432%)
 optimization log               ...        0 min,  0.000 sec (  1.262%)
 hessian update                 ...        0 min,  0.000 sec (  0.171%)
 rational function              ...        0 min,  0.000 sec (  1.425%)

================
 final structure:
================
6
 xtb: 6.2.3 (830e466)
F        -2.70496549785681   -0.15827308555166    0.04708426378741
Cl        2.17225487219378    0.12670830112766   -0.03569780365988
C        -1.10463945001746   -0.06392536108193    0.02054025966543
H        -0.81502984697949   -0.61689835416083   -0.85885449445384
H        -0.79264040442011   -0.51957313506771    0.94555543428654
H        -0.89792967291991    0.99188163473447   -0.03994765962567
 
 Bond Distances (Angstroems)
 ---------------------------
F1-C3=1.6033         C3-F1=1.6033         C3-H4=1.0784         C3-H5=1.0773         C3-H6=1.0776         H4-C3=1.0784         
H5-C3=1.0773         H6-C3=1.0776         
 C  H  Rav=1.0778 sigma=0.0005  Rmin=1.0773  Rmax=1.0784     3
 F  C  Rav=1.6033 sigma=0.0000  Rmin=1.6033  Rmax=1.6033     1
 
 selected bond angles (degree)
 --------------------
H4-C3-F1=104.56                H5-C3-F1=104.48                H5-C3-H4=113.92                H6-C3-F1=104.48                
H6-C3-H4=113.90                H6-C3-H5=114.02                
 
 selected dihedral angles (degree)
 ---------------------------------

 
 CMA Distance (Angstroems)
 ---------------------------
R(CMA):  3.9248
           -------------------------------------------------
          |                Final Singlepoint                |
           -------------------------------------------------

          ...................................................
          :                      SETUP                      :
          :.................................................:
          :  # basis functions                  21          :
          :  # atomic orbitals                  20          :
          :  # shells                           10          :
          :  # electrons                        22          :
          :  max. iterations                   250          :
          :  Hamiltonian                  GFN2-xTB          :
          :  restarted?                      false          :
          :  GBSA solvation                  false          :
          :  PC potential                    false          :
          :  electronic temp.          300.0000000     K    :
          :  accuracy                    1.0000000          :
          :  -> integral cutoff          0.2500000E+02      :
          :  -> integral neglect         0.1000000E-07      :
          :  -> SCF convergence          0.1000000E-05 Eh   :
          :  -> wf. convergence          0.1000000E-03 e    :
          :  Broyden damping             0.4000000          :
          ...................................................

 iter      E             dE          RMSdq      gap      omega  full diag
   1    -13.2311181 -0.132311E+02  0.841E-05    3.65       0.0  T
   2    -13.2311181 -0.312816E-11  0.628E-05    3.65     650.6  T
   3    -13.2311181 -0.461675E-11  0.270E-05    3.65    1514.5  T

   *** convergence criteria satisfied after 3 iterations ***

         #    Occupation            Energy/Eh            Energy/eV
      -------------------------------------------------------------
         1        2.0000           -0.7828198             -21.3016
       ...           ...                  ...                  ...
         5        2.0000           -0.3862285             -10.5098
         6        2.0000           -0.3835609             -10.4372
         7        2.0000           -0.3285279              -8.9397
         8        2.0000           -0.3285014              -8.9390
         9        2.0000           -0.1868279              -5.0838
        10        2.0000           -0.1854303              -5.0458
        11        2.0000           -0.1854300              -5.0458 (HOMO)
        12                         -0.0512466              -1.3945 (LUMO)
        13                          0.2627402               7.1495
        14                          0.3436194               9.3504
        15                          0.3437378               9.3536
        16                          0.3445730               9.3763
       ...                                ...                  ...
        20                          0.3584171               9.7530
      -------------------------------------------------------------
                  HL-Gap            0.1341834 Eh            3.6513 eV
             Fermi-level           -0.1183383 Eh           -3.2201 eV

 SCC (total)                   0 d,  0 h,  0 min,  0.002 sec
 SCC setup                      ...        0 min,  0.000 sec (  3.097%)
 Dispersion                     ...        0 min,  0.000 sec (  0.438%)
 integral evaluation            ...        0 min,  0.001 sec ( 39.084%)
 zeroth order Hamiltonian       ...        0 min,  0.000 sec (  0.816%)
 iterations                     ...        0 min,  0.000 sec ( 14.761%)
 molecular gradient             ...        0 min,  0.001 sec ( 37.710%)
 printout                       ...        0 min,  0.000 sec (  2.855%)

         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         ::                     SUMMARY                     ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::
         :: total energy             -13.174440950682 Eh    ::
         :: gradient norm              0.000999592853 Eh/a0 ::
         :: HOMO-LUMO gap              3.651316999589 eV    ::
         ::.................................................::
         :: SCC energy               -13.231118141341 Eh    ::
         :: -> isotropic ES            0.172030110072 Eh    ::
         :: -> anisotropic ES         -0.001665382287 Eh    ::
         :: -> anisotropic XC          0.002667048510 Eh    ::
         :: -> dispersion             -0.001686547064 Eh    ::
         :: repulsion energy           0.056577075042 Eh    ::
         :: add. restraining           0.000100496723 Eh    ::
         :::::::::::::::::::::::::::::::::::::::::::::::::::::

           -------------------------------------------------
          |                Property Printout                |
           -------------------------------------------------

    * Orbital Energies and Occupations

         #    Occupation            Energy/Eh            Energy/eV
      -------------------------------------------------------------
         1        2.0000           -0.7828198             -21.3016
         2        2.0000           -0.6447373             -17.5442
         3        2.0000           -0.4757169             -12.9449
         4        2.0000           -0.3863563             -10.5133
         5        2.0000           -0.3862285             -10.5098
         6        2.0000           -0.3835609             -10.4372
         7        2.0000           -0.3285279              -8.9397
         8        2.0000           -0.3285014              -8.9390
         9        2.0000           -0.1868279              -5.0838
        10        2.0000           -0.1854303              -5.0458
        11        2.0000           -0.1854300              -5.0458 (HOMO)
        12                         -0.0512466              -1.3945 (LUMO)
        13                          0.2627402               7.1495
        14                          0.3436194               9.3504
        15                          0.3437378               9.3536
        16                          0.3445730               9.3763
        17                          0.3452553               9.3949
        18                          0.3452567               9.3949
        19                          0.3567948               9.7089
        20                          0.3584171               9.7530
      -------------------------------------------------------------
                  HL-Gap            0.1341834 Eh            3.6513 eV
             Fermi-level           -0.1183383 Eh           -3.2201 eV

     #   Z        covCN         q      C6AA      α(0)
     1   9 F      0.725    -0.346    12.864     4.236
     2  17 Cl     0.000    -0.994   126.762    17.135
     3   6 C      3.503     0.047    21.304     6.843
     4   1 H      0.926     0.098     1.808     2.102
     5   1 H      0.926     0.098     1.807     2.102
     6   1 H      0.926     0.098     1.808     2.102

 Mol. C6AA /au·bohr⁶  :        547.014451
 Mol. C8AA /au·bohr⁸  :      15865.128709
 Mol. α(0) /au        :         34.520081


Wiberg/Mayer (AO) data.
largest (>0.10) Wiberg bond orders for each atom
          total WBO             WBO to atom ...
     1  F   0.893        C    3 0.874
     2  Cl  0.012
     3  C   3.836        H    6 0.985    H    5 0.985    H    4 0.985    F    1 0.874
     4  H   0.990        C    3 0.985
     5  H   0.990        C    3 0.985
     6  H   0.990        C    3 0.985

molecular dipole:
                 x           y           z       tot (Debye)
 q only:       -2.873      -0.167       0.047
   full:       -2.666      -0.155       0.043       6.787
molecular quadrupole (traceless):
                xx          xy          yy          xz          yz          zz
 q only:      -25.369      -2.228      12.625       0.636       0.037      12.744
  q+dip:      -27.261      -2.396      13.566       0.683       0.039      13.695
   full:      -26.554      -2.333      13.215       0.667       0.038      13.340

           -------------------------------------------------
          |                Geometry Summary                 |
           -------------------------------------------------

      molecular mass/u    :       69.4858989
   center of mass at/Å    :        0.1414641       0.0082278      -0.0011116
  moments of inertia/u·Å² :        0.3292048E+01   0.3245095E+03   0.3245102E+03
rotational constants/cm⁻¹ :        0.5120713E+01   0.5194804E-01   0.5194793E-01

 * 3 selected distances

     #   Z        #   Z                                     value/Å
     3   6 C      4   1 H                                 1.0784192 (max)
     3   6 C      5   1 H                                 1.0773167 (min)
     3   6 C      6   1 H                                 1.0775510

 * 1 distinct bonds (by element types)

   Z      Z             #   av. dist./Å        max./Å        min./Å
   1 H    6 C           3     1.0777623     1.0784192     1.0773167


optimized geometry written to: xtbopt.xyz


           -------------------------------------------------
          | TOTAL ENERGY              -13.174440950682 Eh   |
          | GRADIENT NORM               0.000999592853 Eh/α |
          | HOMO-LUMO GAP               3.651316999589 eV   |
           -------------------------------------------------

------------------------------------------------------------------------
 * finished run on 2020/04/08 at 23:04:53.779     
------------------------------------------------------------------------
 total:
 * wall-time:     0 d,  0 h,  0 min,  0.033 sec
 *  cpu-time:     0 d,  0 h,  0 min,  0.033 sec
 * ratio c/w:     0.999 speedup
 SCF:
 * wall-time:     0 d,  0 h,  0 min,  0.003 sec
 *  cpu-time:     0 d,  0 h,  0 min,  0.003 sec
 * ratio c/w:     1.000 speedup
 ANC optimizer:
 * wall-time:     0 d,  0 h,  0 min,  0.023 sec
 *  cpu-time:     0 d,  0 h,  0 min,  0.023 sec
 * ratio c/w:     1.000 speedup

